staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1006,6 +1006,10 @@ const (
 // +k8s:openapi-gen=true
 type DriverCache string
 
+//
+// +k8s:openapi-gen=true
+type DriverIO string
```

Comment:
we should add types for DriverIO values, the same way as we do for DriverCache. See a couple of lines below.

Reply:
Added.